kitsune/sumo/jinja2/base.html

```diff
@@ -179,15 +179,15 @@ <h2 class="logo">
     {% endif %}
     <nav id="aux-nav-right" role="navigation">
       <ul>
-        {% if show_fx_download and not user.is_authenticated() %}
+        {% if not user.is_authenticated() %}
```

Comment:
@jpetto Good question. If I understand it correctly, the current logic was to show the Firefox download button _unless_ one or more of these conditions is met:

1. The user agent is already Firefox
2. The person browsing is logged-in
3. The current page is the Firefox product page

This branch abandons the check for the Firefox product page. I presume that check is to avoid the duplicate download buttons, as there's this one in the header, and the Fx product page has another download button.

Here's a screenshot showing both download buttons:
<img width="1151" alt="screenshot 2018-08-20 14 53 07" src="https://user-images.githubusercontent.com/58062/44359129-551b5a00-a48d-11e8-8b72-ada26a670f99.png">

@Dinodog Which of these options should we take:

1. continue to suppress header download button on the Fx product page (where it already has a large download button in the page)
2. keep the header download buttons on all pages (including Fx product pages) and remove the second download button that's part of the Fx product page
3. keep both download buttons on the Fx product page

Now that the visual style of the download button in the header is so prominent (larger and brighter-green), I would lean toward option 2, but I'm happy to implement any of these options.

@jpetto Which do you prefer?

Reply:
I am also in favor of option 2. It maintains consistency, and (IMO) makes the Fx product pages less cluttered/easier to read.

Reply:
@jpetto Agreed with option 2.

Reply:
As per the discussion in the bug, the header firefox button should not show in the `Firefox` product page only. The header button is small compared to the learge download button in Firefox product page.
I am with the option 1 here actually!

Reply:
@safwanrahman Do you still feel this way given the that the update design has a more visually prominent download button in the header? ([screenshot](https://user-images.githubusercontent.com/58062/44359129-551b5a00-a48d-11e8-8b72-ada26a670f99.png))

The header button is still a bit smaller than the 2nd button, but not my as much as the old design.

Reply:
@sgarrity The main issue is, the user can chose which language/system they would like to download. You can see the little items in `System and languages` `whats new`. They are needed feature.
This is available in all other product. Its not appropiate to not show this informations for Firefox product.

Reply:
I'd be nice if `System and languages` and `What's new` were more prominent actually, especially with the more prominant download button in the header.

Reply:
Based on @safwanrahman's point on the `Systems and languages` and `What's new` links, I'm going to switch my recommendation to Option 1 - removing the header button for Fx product pages and continue on with the larger, in-page button.

(FWIW, I also agree with @pmac that it would be best to make those supplemental links more prominent/available with the header button, but that's out of scope here.)